README.md

```diff
@@ -109,6 +110,7 @@ rectors:
         'PHPUnit_Framework_TestCase': 'PHPUnit\Framework\TestCase'
 ```
 
+<a name="rename-partial-namespace"></a>
 ### Replace some part of the namespace
```

Comment:
Love this idea! You can actually use headline links ;)
No need to create a name

Reply:
Oh, didn't know about. Gonna remove those ones I created.

Reply:
It's Github easter egg to make site usability a bit better for prommers.
I them on my blog as well, so you can link to specific headline instead of whole post :+1: